src/uwtools/config/formats/sh.py

```diff
@@ -0,0 +1,80 @@
+import configparser
+from io import StringIO
+from types import SimpleNamespace as ns
+from typing import Optional
+
+from uwtools.config.formats.base import Config
+from uwtools.config.support import depth
+from uwtools.utils.file import OptionalPath, readable, writable
+
+
+class SHConfig(Config):
+    """
+    Concrete class to handle bash config files.
+    """
+
+    def __init__(
+        self,
+        config_file: str,
+    ):
+        """
+        Construct a SHConfig object.
+
+        Spaces should be excluded for bash.
+
+        :param config_file: Path to the config file to load.
+        """
+        super().__init__(config_file)
+        self.parse_include()
+
+    # Private methods
+
+    def _load(self, config_file: OptionalPath) -> dict:
+        """
+        Reads and parses a bash file.
+
+        See docs for Config._load().
+
+        :param config_file: Path to config file to load.
+        """
+        # The protected _sections method is the most straightforward way to get at the dict
+        # representation of the parse config.
+
+        cfg = configparser.ConfigParser()
+        cfg.optionxform = str  # type: ignore
+        sections = cfg._sections  # type: ignore # pylint: disable=protected-access
+        with readable(config_file) as f:
+            raw = f.read()
+        try:
+            cfg.read_string(raw)
+            return dict(sections)
```

Comment:
I'm getting a lack of coverage on this line and I have not been able to figure out why. However, it does look like we have not tested any of the `_load()` functions in any of the formats directly, so that may need to be added

Reply:
I had gone back and forth on that, but I think you're right on this one